.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
Do not push any IDE-dependent files. Please update .gitignore file in the root of the repo to exclude this, see for example .gitignore from other webapp repos.